resources/config.test.edn

```diff
@@ -39,6 +39,7 @@
                                                         :stream-threads-count [1 :int]
                                                         :origin-topic         "topic"
                                                         :upgrade-from         "1.1"
+                                                        :consumer-type        :default
```

Comment:
if a user didn't provide the `:consumer-type` in the config.edn, it will use `:default` as fallback (juts to be backward compatible). We should enforce this on the next major version....